README.md

```diff
@@ -1,68 +1,12 @@
 <!-- badges: start -->
   [![R build status](https://github.com/kelshmo/sageseqr/workflows/R-CMD-check/badge.svg)](https://github.com/kelshmo/sageseqr/actions)
 <!-- badges: end -->
-# sage-seqr
-RNASeq normalization pipeline with R.
+# RNA-seq normalization workflow in R
 
-This repository provides R markdowns for performing covariate adjustments and differential expression analysis of the RNA-seq data from AMP-AD and a Docker environment, thus all package dependencies are loaded. 
+The `sageseqr` package integrates the [`drake` R package](https://github.com/ropensci/drake/), the [`config` package for R](https://cran.r-project.org/web/packages/config/vignettes/introduction.html) and [Synapse](https://www.synapse.org/). `drake` tracks dependency relationships in the workflow and only updates data when it has changed. A `config` file allows inputs and parameters to be explicitly defined in one location. Synapse is a data repository that allows sensitive data to be [stored and shared responsibly](https://docs.synapse.org/articles/article_index.html#governance). 
```

Comment:
The `sageseqr` package integrates the [`drake` R package](https://github.com/ropensci/drake/), the [`config` package for R](https://cran.r-project.org/web/packages/config/vignettes/introduction.html), and [Synapse](https://www.synapse.org/).

Reply:
added a missing comma :)